frontend/pages/blog/index.vue

```diff
@@ -0,0 +1,105 @@
+<template>
+  <main class="py-6 container">
+    <section class="text-center">
+      <h1>Blog</h1>
+      <p class="pt-2">
+        Lorem ipsum dolor sit amet, consetetur sadipscing elitr, sed diam nonumy
+        eirmod tempor invidunt ut labore et dolore magna aliquyam erat, sed diam
+        voluptua. At vero eos et accusam et justo duo dolores et ea rebum.
+      </p>
+    </section>
+
+    <section class="pt-5 row">
+      <div class="col-lg-3 pb-5 pb-lg-0">
+        <h3>Filter</h3>
+      </div>
+
+      <b-overlay
+        :show="isLoading"
+        rounded
+        opacity="0.6"
+        spinner-small
+        spinner-variant="primary"
+        class="col-lg-9 mx-3 mx-md-0"
+      >
+        <template v-if="!isLoading">
+          <p v-if="!blogPosts.length" class="text-muted">
```

Comment:
There should be a distinction between no blog posts from the API and an error thrown during the GET request.